modules/cnf-latency-tests-discovery-mode.adoc

```diff
@@ -8,11 +8,6 @@
 
 Use discovery mode to validate the functionality of a cluster without altering its configuration. Existing environment configurations are used for the tests. The tests can find the configuration items needed and use those items to execute the tests. If resources needed to run a specific test are not found, the test is skipped, providing an appropriate message to the user. After the tests are finished, no cleanup of the preconfigured configuration items is done, and the test environment can be immediately used for another test run.
 
-[IMPORTANT]
-====
-When running the latency tests, **always** run the tests with `-e DISCOVERY_MODE=true` and `-ginkgo.focus` set to the appropriate latency test. If you do not run the latency tests in discovery mode, your existing live cluster performance profile configuration will be modified by the test run.
-====
-
```

Comment:
Discovery mode is set to true by default according to https://github.com/openshift-kni/cnf-features-deploy/pull/1724; we can omit everything mentioning it, especially since we are stating that existing PP is a prerequisite. 
@liornoy may you confirm?

Reply:
Yes, I agree.

Reply:
@liornoy @shajmakh 
Do we want to remove the section [About discovery mode for latency tests](https://69462--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performing-platform-verification-latency-tests.html#discovery-mode_cnf-latency-tests) and its subsection **Limiting the nodes used during tests**?

Reply:
I believe the "limiting the nodes.." doesn't even relate to discovery mode. we can keep it then and move it as a note under "Table 1. Latency test environment variables" since it introduces a way to use another env var for executing the tests on a group of nodes. 
regarding the discovery mode section, I think it should all be removed as it is not relevant any more.

Reply:
I removed the section as discussed and added a new note to retain information from **Limiting the nodes during tetst**.